kivy/core/window/window_x11.pyx

```diff
@@ -210,6 +210,7 @@ class WindowX11(WindowBase):
                 <char *><bytes>title) < 0:
             Logger.critical('WinX11: Unable to create the window')
             return
+        self.window_id = window_id
```

Comment:
Should we declare `window_id` to `WindowBase` as property or attribute? Or declare it in `__init__` of `WindowX11`?
We should have a doc for this public attribute.

Reply:
Well, it could be a variable just for x11. But I think correspondingly, we should test more robustly whether the provider is x11 (currently it's checking if window_id  is defined, which is not robust).

Reply:
Yes, that's what I meant. It could be a property or attribute just for `WindowX11`, but with property we can add the doc.